test/fuzz/transformation_record_synonymous_constants_test.cpp

```diff
@@ -90,13 +90,19 @@ TEST(TransformationRecordSynonymousConstantsTest, IntConstants) {
                                                validator_options);
   ASSERT_TRUE(IsValid(env, context.get()));
 
+#ifndef NDEBUG
   // %3 is not a constant declaration
-  ASSERT_FALSE(TransformationRecordSynonymousConstants(3, 9).IsApplicable(
-      context.get(), transformation_context));
+  ASSERT_DEATH(TransformationRecordSynonymousConstants(3, 9).IsApplicable(
+                   context.get(), transformation_context),
+               "The ids must refer to constants.");
```

Comment:
What changed in this PR to necessitate turning this into a death test?  (Just curious - I couldn't work it out.)

Reply:
There is an assertion in the IsApplicable method checking that the ids refer to constants, I don't know why the test wasn't like this before